chain/src/chain.rs

```diff
@@ -832,7 +873,9 @@ impl BlockChain {
                         &tips_hash, blues
                     );
                     let mut blue_blocks = vec![];
-                    let _selected_parent = blues.remove(0);
+                    let selected_parent = blues.remove(0);
```

Comment:
https://doc.rust-lang.org/std/vec/struct.Vec.html#method.remove
```
Removes and returns the element at position index within the vector, shifting all elements after it to the left.

Note: Because this shifts over the remaining elements, it has a worst-case performance of O(n). If you don’t need the order of elements to be preserved, use [swap_remove](https://doc.rust-lang.org/std/vec/struct.Vec.html#method.swap_remove) instead. If you’d like to remove elements from the beginning of the Vec, consider using [VecDeque::pop_front](https://doc.rust-lang.org/std/collections/struct.VecDeque.html#method.pop_front) instead.
```
Is this place we need check blue_blocks length greater than 1

Reply:
Let me think about it. Maybe it's better to do it.
.